util/inc/leatherman/util/scoped_descriptor.hpp

```diff
@@ -0,0 +1,27 @@
+/**
```

Comment:
Yeah, let's namespace for now. Makes it more obvious to anyone using it that it will just fail on Windows.